### DISCUSSION TITLE
Help Improving Translations / 改进翻译

### DISCUSSION THREAD

**Discussion by chr233** [chr233/ASFEnhance#177] [公告 / Announcement]
https://crowdin.com/project/asfenhance

**Comment by chr233** [comment]
![image](https://github.com/chr233/ASFEnhance/assets/13798576/ffbecb79-4dff-469e-adfe-08d5f9505bba)

If anyone want to improve the translation, left a message below, and I will give you permission

**Comment by VladozPlay** [comment]
I'd be happy to help!

https://crowdin.com/editor/asfenhance/12/zhcn-ru?view=comfortable&filter=basic&value=0

Check pls

**Reply by chr233** [reply]
> I'd be happy to help!
> 
> https://crowdin.com/editor/asfenhance/12/zhcn-ru?view=comfortable&filter=basic&value=0
> 
> Check pls

![image](https://github.com/chr233/ASFEnhance/assets/13798576/6bcec02b-edff-4370-967f-4119ff41995e)
okey
